packages/f36-i18n-utils/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@contentful/f36-i18n-utils",
   "description": "Forma 36 i18n utilities",
-  "version": "1.0.0-alpha.1",
+  "version": "1.0.0-alpha.2",
```

You can revert this change.

For alpha packages, the release is done locally via the `npm run prerelease` command, which automatically bumps the version. I can support you and do it for you once the PR is approved 👍 .

```suggestion
  "version": "1.0.0-alpha.1",
```